collins/collins.go

```diff
@@ -53,7 +53,7 @@ type Error struct {
 	} `json:"data"`
 }
 
-type container struct {
+type Container struct {
```

Comment:
now that you are exporting this type, you should add comments for `godoc` please: http://elliot.land/post/godoc-tips-tricks